user-interface/src/data-verification/transfer/CaseSelection.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render } from '@testing-library/react';
+import { CaseSelection } from './CaseSelection';
+
+describe('Test CaseSelection component', () => {
```

Comment:
We do not appear to be using <CaseSelection> anymore. Delete?

Reply:
I think the plan was to use this component in the modal.